modules/users/client/config/users.authentication.client.routes.js

```diff
@@ -0,0 +1,45 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('users')
+    .config(routeConfig);
+
+  routeConfig.$inject = ['$stateProvider'];
+
+  function routeConfig($stateProvider) {
+    $stateProvider
+      .state('authentication', {
+        abstract: true,
+        url: '/authentication',
+        views: {
+          'main@': {
+            templateUrl: 'modules/core/client/views/layouts/fixed.one.column.client.view.html',
+            controller: 'AuthenticationController'
+          }
+        }
+      })
+      .state('authentication.signup', {
+        url: '/signup',
+        views: {
+          'row-1': {
```

Comment:
@ilanbiala can you give me an example of what you are talking about?  The abstract routes inject layouts into the main ui-view, then the nested routes inject into the layouts views.  I added 4 rows to make it extensive.  More than likely a submodule would use its own layouts, if they want like 6 columns 4 rows or whatever, but providing some basic layouts shouldn't be a huge deal.  Its just HTML that will probably never need updating.

You really cant have only one view since there are 2.  One for the social providers and one for the login form.  I could send the social providers to the hero view but it wouldn't work for the user module as that requires two columns.

Take a look at the most recent commit with the user module.  You have to inject a new layout since the nav has to be beside the actual forms.  So in this case on the abstract routes I put a layout file in the main view and then inject into the layouts file views.  I like doing it this way because it demonstrates that you can do really whatever you want with the views in the submodules.  The way I envision it is we just support a few basic layout HTML files to use in the core and demonstrate how to use them and then the devs can do whatever they want later on in their own modules.